src/Services/GithubApiService.ts

```diff
@@ -146,7 +146,10 @@ export class GithubApiService extends GithubRepository {
         Logger.error(error);
       }
 
-      return new ServiceError("Rate limit exceeded", EServiceKindError.RATE_LIMIT);
+      return new ServiceError(
```

Comment:
This is an unnecessary correction for this PR.

Reply:
Please remove from commit.

Reply:
I think the deno format does that

Reply:
Restored this specific part as original.